concepts/for-loops/links.json

```
@@ -0,0 +1 @@
+[]
```

```suggestion
[]

```

Add a trailing new-line here.